### DISCUSSION TITLE
[Bug]: Is there an example of this layout? What layout should I use?

### DISCUSSION THREAD

**Discussion by ltdys** [antvis/G6#6639] [Q&A]
### Describe the bug / 问题描述

主节点在中心位置 ，子节点在右侧，父节点在左，上，下形成圆弧
![Uploading image2024-11-26 16_11_10.png…]()

### Steps to Reproduce the Bug or Issue / 重现步骤

![image2024-11-26 16_11_10](https://github.com/user-attachments/assets/a8dcb034-353b-4d1b-b5af-e175505c3d23)

### G6 Version / G6 版本

4.x

### OS / 操作系统

- [ ] macOS
- [X] Windows
- [ ] Linux
- [ ] Others / 其他

### Browser / 浏览器

- [X] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari (Limited support / 有限支持)
- [ ] IE (Nonsupport / 不支持)
- [ ] Others / 其他

**Comment by yvonneyx** [comment]
需要提供不同数据量下期望的布局效果，这样我们可以参考并推荐合适的布局
